lib/tennis_game.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+# TennisGame: used to manage the scoring in a tennis game
+class TennisGame
+  TENNIS_POINTS = %w(Love Fifteen Thirty Fourty).freeze
```

Comment:
Would `Advantage` not be a tennis point in this instance? As this would be the case once two players have been on Deuce?